mne/preprocessing/tests/test_ica.py

```diff
@@ -46,8 +46,15 @@
 event_name = op.join(data_dir, 'test-eve.fif')
 test_cov_name = op.join(data_dir, 'test-cov.fif')
 
-ctf_fname = op.join(testing.data_path(download=False), 'CTF',
-                    'testdata_ctf.ds')
+test_base_dir = testing.data_path(download=False)
+ctf_fname = op.join(test_base_dir, 'CTF', 'testdata_ctf.ds')
+
+fif_fname = op.join(test_base_dir, 'MEG', 'sample',
+                    'sample_audvis_trunc_raw.fif')
+eeglab_fname = op.join(test_base_dir, 'EEGLAB', 'test_raw.set')
+eeglab_montage = op.join(test_base_dir, 'EEGLAB', 'test_chans.locs')
+
+ctf_fname2 = op.join(test_base_dir, 'CTF', 'catch-alp-good-f.ds')
```

Comment:
did this indeed have EEG channels?

Reply:
yup. at somepoint I'm going to go through the ctf data and see what it actually is but thats for another day :)